03-transactions.md

```diff
@@ -132,6 +154,19 @@ The output is spent by an input with `nSequence` field set to `1` and witness:
 
 Otherwise, this output is a simple P2WPKH to `remotepubkey`.
 
+##### Leased channel (`option_will_fund`)
+
+In a 'leased' channel, the `to_remote` output that pays the `accepter` node
+is modified so that it is equal to the greater of the
+1 or the `lease_end` - `blockheight`.
```

Comment:
Well, yeah.  If we define lease nomenclature it's easier to change this to say *why*:

`If a lease applies to the channel, the `to_remote` output of the `initiator` ensures the lessor won't get their funds until after lease expiry.`